### DISCUSSION TITLE
Ecologists integration

### DISCUSSION THREAD

**Discussion by dotmobo** [BerriAI/litellm#12137] [General]
Hi guys 

What do you think about an Ecologists integration in the admin interface to monitor energy consumption and environmental footprint of LLMs ?

Cf : https://github.com/genai-impact/ecologits

Thanks

**Comment by flefevre** [comment]
It will be great to integrate it!

With the growing environmental concerns around the energy use of large AI models, more teams are looking for ways to track and reduce their carbon impact. Ecologits provides a lightweight and easy-to-integrate tool that estimates energy consumption and associated CO₂ emissions of code execution. Adding this capability to Langfuse would:

- Empower users to better understand the environmental impact of their LLM usage.
- Provide a competitive advantage by promoting sustainable AI practices.
- Align Litellm with emerging responsible AI principles.

**Comment by flefevre** [comment]
![image](https://github.com/user-attachments/assets/d78a96a5-c9c4-4ab1-aab0-f9b3ddd9a016)
